src/contexts/SetComponents/SetComponentsProvider.tsx

```diff
@@ -268,6 +222,44 @@ const SetComponentsProvider: React.FC = ({ children }) => {
     eth2xflipPrice,
   ])
 
+  useEffect(() => {
+    if (
+      chainId &&
+      chainId === POLYGON_CHAIN_DATA.chainId &&
+      provider &&
+      dpiTokenPolygonAddress &&
+      mviTokenPolygonAddress &&
```

Comment:
Copied from old code but not sure if `dpiTokenPolygonAddress` and `mviTokenPolygonAddress` actually need to be part of the check.